app/views/admin/move_advisers/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for(@form, url: choose_to_firm_admin_move_advisers_path, method: :get) do |f| %>
+  <h1>Choose advisers to move</h1>
+  <% if @form.errors.any? %>
+    <ul class="t-errors bg-danger">
+      <% @form.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <div class="form-group">
+    <%= f.collection_check_boxes(:adviser_ids, @form.from_firm.advisers.order(:reference_number),
+          :id, ->(adviser) { "(#{adviser.reference_number}) #{adviser.name}" }) do |b| %>
```

Comment:
The defining of a stabby proc here for the checkbox label is interesting. I've not seen that done before. You could add a method to adviser which does this `:ref_number_and_name` or something, but then I guess you're then polluting the model with decoration, so it's deciding what's the lesser of two evils. I can read and understand it as it stands so not suggesting you actually do that refactor.